conda_smithy/templates/github-actions.tmpl

```diff
@@ -14,7 +14,13 @@ name: Build conda package
 {%- if github_actions.self_hosted %}
 on: [push]
```

Comment:
Fix this too?

Reply:
But in this case there's no trigger duplication because pull_requests are not enabled. I am assuming people might still want to run CI there, just without uploads.